src/styles/class-overrides.scss

```diff
@@ -99,51 +89,93 @@
 
 .usa-footer__secondary-section {
   color: white;
-  background-color: color("base-darker");
+  background-color: color("base-dark");
   .usa-link,
   .usa-link--external {
     color: white;
   }
 }
 
-.language-selector__nav > .usa-header--extended {
-  padding-bottom: 0rem;
-  padding-top: 0rem;
-}
-
-//Custom class names:
-
-.banner,
-.banner__logo-title {
+.banner {
   display: flex;
   align-items: center;
   justify-content: space-between;
-  .usa-accordion__button,
-  .usa-nav__link {
-    color: white;
+  height: fit-content;
+  .usa-accordion__button {
+    background-image: none;
+    padding: 0 units("105");
+    p {
+      padding-left: units("105");
+    }
+  }
+  .usa-accordion__button[aria-expanded="false"] {
+    color: color("base-dark");
     background-color: color("primary");
-    button[aria-expanded="true"] {
-      background-color: color("base-darker");
+    p {
+      color: color("base-dark");
     }
   }
+  .usa-accordion__button[aria-expanded="true"] {
+    color: white;
+    background-color: color("primary");
+  }
+}
+
+.banner__logo-title {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 }
 
 .banner__language-selector-label {
   display: flex;
   align-items: center;
   justify-content: space-between;
-  p {
-    padding-left: 10px;
+}
+
+.usa-button--unstyled {
+  color: white;
+  text-decoration: none;
+}
+
+.banner__language-selector-item--RtoL {
+  display: flex;
+  flex-direction: row-reverse;
+  .usa-button--unstyled {
+    &:hover {
+      color: white;
+      text-decoration: none;
+      background-color: color("base-dark");
+      padding-right: units("05");
+      border-right: units("05") solid color("primary");
+    }
   }
 }
 
-.banner__language-selector-container {
-  .usa-nav__submenu .usa-nav__submenu-item a {
-    text-decoration: none;
+.banner__language-selector-item--LtoR {
+  .usa-button--unstyled {
+    &:hover {
+      color: white;
+      text-decoration: none;
+      background-color: color("base-dark");
+      padding-left: units("05");
+      border-left: units("05") solid color("primary");
+    }
+  }
+}
+
+.banner__language-selector-button-group {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  .usa-button--unstyled {
+    padding: units("05");
+    margin: 0 units("05");
     &:hover {
-      // text-decoration: none;
-      padding: $mocto-link-padding;
-      border-left: $mocto-link-border-width solid color("primary");
+      color: white;
```

Comment:
In our coding agreement, we should set styling conventions to follow.

Reply:
Yes definitely!